lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,646 @@
+
+        return exec_sql(self, ' '.join(query), ddl=True)
+
+    def set_schema(self):
+        """Implements ALTER SEQUENCE SET SCHEMA command behavior."""
+        query = ['ALTER SEQUENCE']
+        query.append(self.__add_schema())
+        query.append('SET SCHEMA %s' % pg_quote_identifier(self.module.params['newschema'], 'schema'))
+
+        return exec_sql(self, ' '.join(query), ddl=True)
+
+    def __add_schema(self):
+        return '.'.join([pg_quote_identifier(self.schema, 'schema'),
+                        pg_quote_identifier(self.name, 'sequence')])
+
+
+# ===========================================
+# Module execution.
+#
+
+def main():
+    argument_spec = postgres_common_argument_spec()
+    argument_spec.update(
+        sequence=dict(type='str', required=True, aliases=['name']),
+        state=dict(type='str', default='present', choices=['absent', 'present']),
+        data_type=dict(type='str', choices=['bigint', 'integer', 'smallint']),
+        increment=dict(type='int'),
+        minvalue=dict(type='int', aliases=['min']),
+        maxvalue=dict(type='int', aliases=['max']),
+        start=dict(type='int'),
+        cache=dict(type='int'),
+        cycle=dict(type='bool'),
+        schema=dict(type='str', default='public'),
+        cascade=dict(type='bool'),
+        rename_to=dict(type='str'),
+        owner=dict(type='str'),
+        newschema=dict(type='str'),
+        db=dict(type='str', default='', aliases=['login_db', 'database']),
+        session_role=dict(type='str'),
+    )
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        mutually_exclusive=[
+            ['absent', 'data_type'],
+            ['absent', 'increment'],
+            ['absent', 'minvalue'],
+            ['absent', 'maxvalue'],
+            ['absent', 'start'],
+            ['absent', 'cache'],
+            ['absent', 'cycle'],
+            ['absent', 'rename_to'],
+            ['absent', 'owner'],
+            ['absent', 'newschema'],
+            ['rename_to', 'data_type'],
+            ['rename_to', 'increment'],
+            ['rename_to', 'minvalue'],
+            ['rename_to', 'maxvalue'],
+            ['rename_to', 'start'],
+            ['rename_to', 'cache'],
+            ['rename_to', 'cycle'],
+            ['rename_to', 'cascade'],
+            ['rename_to', 'owner'],
+            ['rename_to', 'newschema'],
+            ['cascade', 'data_type'],
+            ['cascade', 'increment'],
+            ['cascade', 'minvalue'],
+            ['cascade', 'maxvalue'],
+            ['cascade', 'start'],
+            ['cascade', 'cache'],
+            ['cascade', 'cycle'],
+            ['cascade', 'owner'],
+            ['cascade', 'newschema'],
+        ]
+    )
+
+    # Note: we don't need to check mutually exclusive params here, because they are
+    # checked automatically by AnsibleModule (mutually_exclusive=[] list above).
+
+    # sequence = module.params["sequence"]
```

Comment:
Are these lines needed?

Reply:
Sorry, no not anymore. I will remove it.